pkg/bitcoin/htlc/database/connection.ts

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config(({ path: '/Users/madsjordt/Desktop/Projects/AMOS/swapchain/.env' }))
+import server from "./server"
+import mongoose from "mongoose"
+ 
+  const port = process.env.PORT || 8080
+  const mongoPort = process.env.MONGO_PORT || 27017
+  const dbName = process.env.DB_NAME || "test"
+  const URI = process.env.MONGOURI
+
+  mongoose
+    .connect(URI!, { useUnifiedTopology: true, useNewUrlParser: true }).then(() => {
+        console.log("MongoDB connected")
+        server.listen(port, () => {
```

Comment:
this is the file for connecting to mongo, not starting your API server, please use an index.ts file for that.
 
Index.ts is also ignored in coverage because only code that is not testable should go in there.